client/main.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import App from './application'
+import Store from './store'
+import io from 'socket.io-client'
+
+const socket = io.connect('http://localhost:3000')
```

Comment:
you should be able to type in just the root url here: `io.connect('/')`. Can you give that a shot?